src/osd/scrubber/pg_scrubber.h

```diff
@@ -573,16 +574,28 @@ class PgScrubber : public ScrubPgIF,
   //  artificial PG vs. PrimaryLogPG distinction. Some of the services used
   //  by the scrubber backend are PrimaryLog-specific.
 
-  virtual void add_to_stats(const object_stat_sum_t& stat)
+  void add_to_stats(const object_stat_sum_t& stat) override
   {
     ceph_assert(0 && "expecting a PrimaryLogScrub object");
   }
 
-  virtual void submit_digest_fixes(const digests_fixes_t& fixes)
+  void submit_digest_fixes(const digests_fixes_t& fixes) override
   {
     ceph_assert(0 && "expecting a PrimaryLogScrub object");
   }
 
+  /// \todo clearly identify when to use which of the following two:
+  CephContext* get_pg_cct() const override { return m_pg->cct; }
+  CephContext* get_osd_cct() const override { return m_pg->cct; }
```

Comment:
As far as I know, it is not possible for those to be different.  The CephContext* instance should be process-wide.

Reply:
Thanks. Will merge.